resources/css/components/cards.css

```diff
@@ -10,6 +10,39 @@
     .badge {
       @apply absolute left-4 top-4 z-2;
     }
+    &-carousel{
+      @apply relative h-full;
+
+      carousel-slide{
+        @apply inline-block h-full opacity-0 transition-all;
+        &.active{
+          @apply opacity-100;
+        }
+        img{
+          @apply absolute top-0;
+        }
+      }
+      &-navigation{
+        @apply absolute top-1/2 w-full;
+        a{
+          @apply bg-white dark:bg-dark-600 bg-opacity-50 dark:bg-opacity-50 rounded-full transition-all absolute -translate-y-1/2;
+          &:hover,
+          &:focus{
+            @apply bg-opacity-80 dark:bg-opacity-80
+          }
+        }
+        &-next{
+          @apply left-1;
+        }
+        &-prev{
+          @apply right-1;
+        }
+      }
+      &-slide-count{
+        @apply absolute bottom-0 w-full text-center bg-white bg-opacity-50 dark:bg-opacity-50 dark:bg-dark-600;
```

Comment:
```suggestion
        @apply absolute bottom-0 w-full text-center bg-white bg-opacity-50 dark:bg-opacity-50 dark:bg-dark-600 text-[.75rem] z-10;
```

![Снимок экрана 2024-06-04 в 11 52 41](https://github.com/moonshine-software/moonshine/assets/12373059/30114a9c-54ea-44e7-a048-1058d86e20a5)

стили просто для примера